### PR TITLE
[CI] Add docker image build and push pipeline

### PR DESCRIPTION
**Pull request by look4regev**
Built and pushed successfully when I used a test branch:
https://app.travis-ci.com/github/hiredscorelabs/cornell/jobs/546283438
https://hub.docker.com/repository/docker/hiredscorelabs/cornell

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,12 @@ test:
 configure:
 	pip install -e .'[dev]'
 
+docker_build_push:
+	docker build . -t hiredscorelabs/cornell:latest
+	echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
```
Comment (look4regev) [inline Makefile:10]: Those were stored on the travis project settings of course, accessible only to master branch, using a token of our hiredscorelabs docker hub user.

File: .travis.yml [hunk 2]
```diff
@@ -7,3 +7,13 @@ install:
   - make configure
 script:
   - make test
+
+jobs:
+  include:
+    - stage: deploy
+      python: 3.9
```
Comment (look4regev) [inline .travis.yml:14]: this promises the deploy step will happen only once and not for every python version.

**Review by yaelmi3 — APPROVED** [review]
Not all heroes wear capes ❤️

File: Makefile [hunk 3]
```diff
@@ -5,7 +5,20 @@ test:
...
\ No newline at end of file
+	twine upload dist/*
+
+docker_tag_push:
+	REPO="hiredscorelabs/cornell"
+	TAG="${REPO}:$(python cornell/_version.py)"
+	docker pull ${REPO}:latest
+	docker tag ${REPO}:latest ${TAG}
+	echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
+	docker push ${TAG}
```
Comment (look4regev) [inline Makefile:24]: Tested successfully from local:
https://hub.docker.com/layers/hiredscorelabs/cornell/1.0.7/images/sha256-2e5d80469449a6f89e9144512cb3636af860e10890f75775878f13d62b0dfd65?context=explore

**Review by AvnerCohen — APPROVED** [review]
👍

**Pull request merged**